kobo/settings/base.py

```diff
@@ -1363,6 +1363,9 @@ def dj_stripe_request_callback_method():
     'default': env.cache(default='redis://redis_cache:6380/3'),
 }
 
+# How long to retain cached responses for kpi endpoints
+ENDPOINT_CACHE_DURATION = env.str('ENDPOINT_CACHE_DURATION', 60 * 15)  # 15 minutes
```

Comment:
I think we should better use `env.int()` instead of `env.str()` here? Even the default value is an integer?